### DISCUSSION TITLE
[Help] – TailwindCSS doesn't work after deployment to the Vercel

### DISCUSSION THREAD

**Discussion by kkdima** [tailwindlabs/tailwindcss#5748] [Help] [answered]
When I deploy my NextJS project to Vercel I can see only rendered HTML align to the left side of the screen.
For some reason deployment dropping all tailwind styles.

I tried to turn off purge and it still didn't help. I don't think turning off purge is a good practice anyway, so I'm looking for a better solution.
Is it something wrong in my config?
Maybe nextjs config? Am I missing something?

[Link to the repo: 
](https://github.com/kkdima/nike-homepage/tree/main)
Here's my `tailwind.config.js`:
```
module.exports = {
	mode: 'jit',
	purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
	darkMode: false, // or 'media' or 'class'
	theme: {
		extend: {
			maxWidth: {
				// prettier-ignore
				'880': '880px',
			},
			spacing: {
				// prettier-ignore
				'13': '3.25rem',
				// prettier-ignore
				'880': '880px',
			},
			gridTemplateColumns: {},
			gridTemplateRows: {
				// Complex site-specific row configuration
				two: 'min-content 1fr',
			},
			colors: {
				'color-1': '#F8F8F8',
				'nike-light-gray': '#757575',
				'nike-ad': '#e7e7e7',
				'nike-white': '#FFFFFF',
				'nike-black': '#111111',
			},
			height: {
				// prettier-ignore
				'xxll': '550px',
			},
			fontFamily: {
				// prettier-ignore
				'futura': ['Futura-Bold'],
				// prettier-ignore
				'futura-condensed': ['Futura Condensed'],
				'helvetica-neue-medium-cond': ['HelveticaNeue MediumCond'],
				'helvetica-neue-medium': ['HelveticaNeue Medium'],
				'helvetica-neue-regular': ['HelveticaNeue Regular'],
				// prettier-ignore
				'tradegothic': ['TradeGothic LT'],
			},
		},
	},
	plugins: [require('@tailwindcss/forms')],
};

```

**Comment by simonswiss** [comment, marked as answer]
Hey!

Is there still an issue, or did this commit fix it?

https://github.com/kkdima/nike-homepage/commit/73ed027af3394207daff91b1b4ad2b8e2f3cd1bb

I cloned your repo locally and it looks just like what you have deployed on the demo URL.

By the way, when using the `src` folder, you can also put your `pages` directory in there (but you don't have to), and Next.js will pick it up ok:

https://nextjs.org/docs/advanced-features/src-directory

Let me know if you still need help on this 👍

**Reply by kkdima** [reply]
Thank you for your reply! 
I updated the repo with a solution for that error.
The mistake was on the 3rd line of `tailwind.config.js `. It was the wrong path to look for files when purge.
**Correct line:** `	purge: [
		'./pages/**/*.{js,ts,jsx,tsx}',
		'./src/components/**/*.{js,ts,jsx,tsx}',
	],`

**Reply by zelal-dev** [reply]
You saved my time. Thank you. It works for me

**Reply by ashutosh86941** [reply]
> You saved my time. Thank you. It works for me

hey zelal, can you tell me how to do it, i am still not able to understand why tailwind is not applying in my project after deploying it on github.

**Reply by ashutosh86941** [reply]
> Hey!
> 
> Is there still an issue, or did this commit fix it?
> 
> [kkdima/nike-homepage@73ed027](https://github.com/kkdima/nike-homepage/commit/73ed027af3394207daff91b1b4ad2b8e2f3cd1bb)
> 
> I cloned your repo locally and it looks just like what you have deployed on the demo URL.
> 
> By the way, when using the `src` folder, you can also put your `pages` directory in there (but you don't have to), and Next.js will pick it up ok:
> 
> https://nextjs.org/docs/advanced-features/src-directory
> 
> Let me know if you still need help on this 👍

hey simon can you tell me why my tailwind css is not applied in my project when i deployed it on github, can you clone my project and resolve my issue, i have wasted so much time already,  problem is in my fb-login-page-clone-project, you can check it in my reposatory, everything is working in vite(local host) dist file is also working in live server but when i am deploying my project on github using postcss (dist file) the tailwind is not applying.
